src/Illuminate/Support/Str.php

```diff
@@ -774,7 +774,7 @@ public static function padBoth($value, $length, $pad = ' ')
     public static function padLeft($value, $length, $pad = ' ')
     {
         if (function_exists('mb_str_pad')) {
-            return mb_str_pad($value, $length, $pad, STR_PAD_LEFT);
+            return mb_str_pad((string)$value, $length, $pad, STR_PAD_LEFT);
```

Comment:
Maybe instead of casting to `string`, we should do `$value ?? ''`?

Reply:
Agree. I did this way because is the same is did in Stringable's constructor (https://github.com/laravel/framework/blob/e574ffa03436a85d0aefccca9d8127746b3032aa/src/Illuminate/Support/Stringable.php#L33) and is the same purpose, for me